### PR TITLE
Add a crate for logging directly to journald

### PR DESCRIPTION
**Pull request by tmccombs**

### CODE REVIEW

File: crates/journald/lib.rs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    type Error = ::Error;
+    fn log(&self, info: &Record, logger_values: &OwnedKeyValueList) -> Result<(), ::Error> {
+        let mut serializer = Serializer::new();
+        serializer.add_field("PRIORITY", level_to_priority(info.level()));
+        serializer.add_field("MESSAGE", info.msg());
+        serializer.add_field("CODE_FILE", info.file());
+        serializer.add_field("CODE_LINE", info.line());
+        serializer.add_field("CODE_MODULE", info.module());
+        serializer.add_field("CODE_FUNCTION", info.function());
+        for &(ref k, ref v) in logger_values.iter() {
```
Comment (dpc) [inline crates/journald/lib.rs]: I have broke this in master (soon to be released as slog 1.1), see CHANGELOG.md. Sorry for that - it was for a good reason, and I think this is the first 3rd-party serializing drain. Can you try against `master` branch?

**Comment by dpc** [comment]
Awesome. Sorry for troubles with breaking changes on master (see other comment).

BTW. I can merge it to slog, but I am thinking about creating `slog` organization and start moving things there. This could be a crate hosted there. What do you think?

**Comment by tmccombs** [comment]
> I am thinking about creating slog organization and start moving things there. This could be a crate hosted there. What do you think?

That sounds fine to me.

**Comment by tmccombs** [comment]
In fact it might be good to have  a seperate repo for this, since it requires libsystemd, so the travis build will need to run on trusty.

**Comment by dpc** [comment]
I've created https://github.com/slog-rs/journald and you should have admin rights. I have never organized the community on github, so in case anything is wrong, please let me know.

I'll close this issue for now, and thank you so much again for this PR! :)

**Pull request closed without merging**